package.json

```diff
@@ -37,13 +39,16 @@
   "dependencies": {
     "@grpc/proto-loader": "^0.4.0",
     "body-parser": "^1.18.3",
+    "elastic-apm-node": "^1.14.5",
     "express": "^4.16.4",
-    "grpc": "^1.19.0",
+    "grpc": "^1.20.3",
     "helmet": "^3.14.0",
     "jayson": "^2.1.1",
     "jsonwebtoken": "^8.4.0",
     "morgan": "^1.9.1",
     "passport": "^0.4.0",
-    "passport-saml": "^1.0.0"
+    "passport-saml": "^0.33.0",
+    "winston": "^3.2.1",
```

Comment:
Unused dependency, remove it and `winston-elasticsearch`, we even said that we will not use it.